pkg/export/export.go

```diff
@@ -397,7 +402,7 @@ func (e *Exporter) Export(metadata MetadataFunc, batch []record.RefSample) {
 	e.mtx.Unlock()
 
 	if !ok {
-		prometheusSamplesDiscarded.WithLabelValues("no-ha-range").Inc()
+		prometheusSamplesDiscarded.WithLabelValues("no-ha-range").Add(float64(batchSize))
```

Comment:
should this be `prometheusSamplesDropped` rather than `prometheusSamplesDiscarded`

Reply:
Yes - done.